Unable to index some datasets from csv connector

### Problem description

When I use /auth/admin endpoint to create a csv connector, using file of magda-csv-connector/src/test/csv3.xlsx, a connector is successfully created and can start creating datasets in the registry. However, the indexer will throw exception
```
[ERROR] [06/27/2019 06:52:01.226] [indexer-akka.actor.default-dispatcher-7] [akka.actor.ActorSystemImpl(indexer)] Exception encountered
spray.json.DeserializationException: Expected String as JsString, but got ["AI"]
        at spray.json.package$.deserializationError(package.scala:23)
        at spray.json.ProductFormats$class.fromField(ProductFormats.scala:63)
        at au.csiro.data61.magda.indexer.external.registry.WebhookApi.fromField(WebhookApi.scala:21)
        at spray.json.ProductFormatsInstances$$anon$6.read(ProductFormatsInstances.scala:167)
        at spray.json.ProductFormatsInstances$$anon$6.read(ProductFormatsInstances.scala:149)
        at spray.json.JsValue.convertTo(JsValue.scala:33)
        at au.csiro.data61.magda.model.Registry$RegistryConverters$$anonfun$41.apply(Registry.scala:391)
        at au.csiro.data61.magda.model.Registry$RegistryConverters$$anonfun$41.apply(Registry.scala:391)
        at scala.Option.map(Option.scala:146)
        at au.csiro.data61.magda.model.Registry$RegistryConverters$class.convertRegistryDataSet(Registry.scala:391)
        at au.csiro.data61.magda.indexer.external.registry.WebhookApi.convertRegistryDataSet(WebhookApi.scala:21)
        at au.csiro.data61.magda.indexer.external.registry.WebhookApi$$anonfun$1$$anonfun$apply$1$$anonfun$7$$anonfun$8.apply(WebhookApi.scala:65)
        at au.csiro.data61.magda.indexer.external.registry.WebhookApi$$anonfun$1$$anonfun$apply$1$$anonfun$7$$anonfun$8.apply(WebhookApi.scala:64)
        at scala.collection.immutable.List.map(List.scala:284)
        at au.csiro.data61.magda.indexer.external.registry.WebhookApi$$anonfun$1$$anonfun$apply$1$$anonfun$7.apply(WebhookApi.scala:64)
        at au.csiro.data61.magda.indexer.external.registry.WebhookApi$$anonfun$1$$anonfun$apply$1$$anonfun$7.apply(WebhookApi.scala:61)
        at au.csiro.data61.magda.indexer.external.registry.WebhookApi$$anonfun$1$$anonfun$apply$1$$anonfun$9.apply(WebhookApi.scala:75)
        at au.csiro.data61.magda.indexer.external.registry.WebhookApi$$anonfun$1$$anonfun$apply$1$$anonfun$9.apply(WebhookApi.scala:75)
        at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:253)
        at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
        at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:36)
        at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:55)
        at akka.dispatch.BatchingExecutor$BlockableBatch$$anonfun$run$1.apply$mcV$sp(BatchingExecutor.scala:91)
        at akka.dispatch.BatchingExecutor$BlockableBatch$$anonfun$run$1.apply(BatchingExecutor.scala:91)
        at akka.dispatch.BatchingExecutor$BlockableBatch$$anonfun$run$1.apply(BatchingExecutor.scala:91)
        at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:72)
        at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:90)
        at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:40)
        at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:44)
        at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
        at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
        at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
        at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
```

### Problem reproduction steps
See above


### Screenshot / Design / File reference
Jack identified the problem:
![image.png](https://images.zenhubusercontent.com/5bd00412ff86694208090a3e/8f2bfde3-a953-4804-9738-29f918886793)

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -7,6 +7,7 @@ Search:
 Indexer:
 
 -   Fixed indexer throws an error when temporalCoverage aspects intervals is an empty array
+-   Fixed indexer throws an error when affiliatedOrganisation field is created
 
 Registry:
 

--- a/magda-scala-common/src/main/scala/au/csiro/data61/magda/model/Misc.scala
+++ b/magda-scala-common/src/main/scala/au/csiro/data61/magda/model/Misc.scala
@@ -62,7 +62,7 @@ package misc {
      sourceSystem: Option[String] = None,
      likelihoodOfRelease: Option[String] = None,
      isOpenData: Option[Boolean] = None,
-     affiliatedOrganisation: Option[String] = None)
+     affiliatedOrganisation: Option[Seq[String]] = None)
 
   case class AccessControl(
      ownerId: Option[String] = None,

